scripts/linters/pylint_extensions_test.py

```diff
@@ -24,7 +24,7 @@
 import tempfile
 import unittest
 
-from core import python_utils
+from core import utils as Utils
```

Comment:
Why `as Utils`?

Reply:
Ah, I see, please rather modify the line at the bottom to something like `from pylint import utils as pylint_utils # isort:skip`

Reply:
sure !!

Reply:
PTAL @vojtechjelinek